### DISCUSSION TITLE
Plot tags with colors as a multiline plot on stats page

### DISCUSSION THREAD

**Discussion by aditya95sriram** [monkeytypegame/monkeytype#4495] [Ideas] [answered]
In case multiple tags are selected, it would be cool if the main plot/chart/graph plots the different tags with different colors. For example, if I have two tags `work` and `home`, maybe instead of being plotted as a single yellow line (assuming Serika theme), they could be plotted using blue and red (say) respectively. That way the plot conveys much more information.

To clarify even further, here's how it looks now:

![image](https://github.com/monkeytypegame/monkeytype/assets/5451736/b1427e26-bb04-4daa-8dec-16916d1acdf8)

and here's how it could look (this image doesn't use real data, it's just for demonstration)

![image](https://farm8.staticflickr.com/7420/10939727363_e2bae9c236_b.jpg)

(source: https://stackoverflow.com/a/20064690/1614140)

**Comment by Miodec** [comment, marked as answer]
It would be difficult to make it work in every theme

**Reply by aditya95sriram** [reply]
Hmm, fair point. How about reusing the colors from the "burst heatmap"?

![image](https://github.com/monkeytypegame/monkeytype/assets/5451736/b7c8e78a-23c6-4738-883d-54658454c764)
(theme: solarized dark)

**Reply by Miodec** [reply]
Well, thats only 5 colors.. you can have more than 5 tags.

**Reply by aditya95sriram** [reply]
Ah ok, got it
